lib/pipeline.ts

```diff
@@ -153,13 +154,48 @@ export interface PipelineProps {
   chimeMessage?: string;
 }
 
+export interface MergeBackStage {
+
+  /**
+   * Which stage should the merge back be part of. (Created if missing)
+   *
+   * @default 'MergeBack'
+   */
+  readonly name?: string
+
+  /**
+   * The name of the stage that the merge back stage should go after of. (Must exist)
+   */
+  readonly after: string;
+}
+
+/**
+ * Options for configuring an auto merge-back for this pipeline.
+ */
+export interface AutoMergeBackOptions extends Omit<AutoMergeBackProps, 'repo'> {
+
+  /**
+   * Specify stage options to create the merge back inside a stage of the pipeline.
+   *
+   * @default - The CodeBuild project will be created indepdent of any stage.
+   */
+  readonly stage?: MergeBackStage
+}
+
+/**
+ * Options for configuring an auto bump for this pipeline.
+ */
+export interface AutoBumpOptions extends Omit<AutoBumpProps, 'repo'> {
```

Comment:
how does jsii handle this?

Reply:
Well, delivlib isn't a `jsii` module...

Reply:
To answer @eladb's direct question - https://github.com/aws/jsii/issues/1707

Reply:
> Well, delivlib isn't a jsii module...

Nevertheless, this feels very backward.
Are we not able to construct these forward, rather than omit backwards?

Reply:
Start by defining `AutoPullRequestOptions`, then define `AutoMergeBackProps` based on that, etc. etc.